JSDemos/Demos/PivotGrid/ChartIntegration/Angular/description.md

```diff
@@ -0,0 +1,44 @@
+
+## Convert PivotGrid Fields to Chart Series
+You can use the following **integrationOptions** to specify how PivotGrid fields should be converted to chart series:
+
+- **alternateDataFields**: Boolean       
+Specifies whether to alternate data fields in the resulting chart. This demo sets the **alternateDataFields** field to **false**.
+
+- **dataFieldsDisplayMode**: *"singleAxis"* | *"splitAxes"* | *"splitPanes"*     
+Accepts one of the following values.   
+
+    - *"singleAxis"* - values of all data fields are displayed on a single [value axis](/Documentation/ApiReference/UI_Components/dxChart/Configuration/valueAxis/).  
+    - *"splitAxes"* - each data field gets an individual value axis. Applies only if **putDataFieldsInto** is *"series"*.  
+    - *"splitPanes"* - each data field gets an individual [pane](/Documentation/ApiReference/UI_Components/dxChart/Configuration/panes/). Applies only if **putDataFieldsInto** is *"series"*.    
+
+    Applies only if the PivotGrid contains several [data fields](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/). In this demo, the **dataFieldsDisplayMode** property's value is *"splitPanes"*.
+
+- **inverted**: Boolean      
+If **true**, [row field](/Documentation/Guide/UI_Components/PivotGrid/Visual_Elements/#Headers) values go to the [chart arguments](/Documentation/ApiReference/UI_Components/dxChart/Configuration/argumentAxis/) and [column field](/Documentation/Guide/UI_Components/PivotGrid/Visual_Elements/#Headers) values form [series](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/). If **false** - vice versa.
+
+- **putDataFieldsInto**: *"series"* | *"args"*     
+Accepts one of the following values.
+
+    - *"args"* - data fields go to the Chart arguments.  
+    - *"series"* - data fields go to the Chart series.
+
+## Advanced Customization
+The **integrationOptions** object also can contain the following functions that allow you to process PivotGrid data before visualization and customize the Chart and its series:
+
+- **processCell**       
+Processes a PivotGrid cell's data before the Chart displays it.
+
+- **customizeChart**        
+Allows you to customize the Chart's appearance.
+
+- **customizeSeries**       
+Allows you to customize the Chart's series.
+
+Refer to the **bindChart(chart, integrationOptions)** method's [description](/Documentation/ApiReference/UI_Components/dxPivotGrid/Methods/#bindChartchart_integrationOptions) for more information on these functions. 
```

Comment:
You can place a link directly on the method's name to save space and simplify your sentence:

```suggestion
Refer to the [bindChart(chart, integrationOptions)](/Documentation/ApiReference/UI_Components/dxPivotGrid/Methods/#bindChartchart_integrationOptions) method description for more information on these functions.
```